common/MathLibrary/MathHelper.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace MathLibrary
+{
+    public class MathHelper
```

Comment:
Do you require it for layered architecture?
It could be separate project or you could even remove it

Reply:
just lazy